features/mbedtls/VERSION.txt

```diff
@@ -1 +1 @@
-mbedtls-2.15.1
```

Comment:
This must be `development` not `development-proposed`. That's just a transient branch used for testing. This must be changed.